README.md

```diff
@@ -14,18 +14,31 @@ before. Once you're good, import this module and pass the appropriate variables.
 ## Example Usage
 
 ```hcl-terraform
+module "default_cache_behavior" {
+  source  = "7Factor/s3-website/aws//modules/cache_behavior"
+  version = "~> 2"
```

Comment:
I'm not entirely sure how versioning works on these submodules. I think we need to try it and find out.